_guides/identdfile.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: documentation
+title: Identify users with identd or oident
+---
+
+When the IRC server gets a connection request, it will query your system for the ident.
+An ident server can read the ident file and send this ident back to the IRC server.
+
+## Built-in identd server
+
+The identd option is a lightweight server implemented in The Lounge itself.
+On Linux, this currently requires The Lounge to run as root to be able to bind to port 113.
+
+As such, this is not really recommended and `oidentd` is a better choice.
```

Comment:
If in our own documentation, we recommend not using `identd`, do we want to remove its support altogether in the future?

Reply:
It's actually a solid solution with oidentd forwarding as it solves multiple problems.

Reply:
Gotcha, I didn't realize that when reading the document. The way I read it, it simply said indentd is discouraged, and I didn't catch that when combined with oidentd it becomes much better.